scripts/generateExpoGoIntermediateCertificate.ts

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs/promises';
```

Comment:
Fine to leave as-is but in scripts I more often use the sync fs methods. They're faster than their async variants and easier to debug and read at the cost of disallowing concurrency, which tends not to matter in most scripts anyway.